src/main/java/seedu/storemando/model/item/Quantity.java

```diff
@@ -31,7 +31,13 @@ public Quantity(String quantity) {
      * Returns true if a given string is a valid quantity number.
      */
     public static boolean isValidQuantity(String test) {
-        return test.matches(VALIDATION_REGEX);
+        if (test.matches(VALIDATION_REGEX)) {
```

Comment:
I think if you can avoid nesting if-else it would be great. So you can do
(test.matches(VALIDATION_REGEX) && value > 0) {
return true;
}

It increases readability as well.

Reply:
Alright, I will update it respectively.

Reply:
But as long as your Long.parseLong(test) is the second condition to test.matches(VALIDATION_REGEX), it will be fine due to short-circuit.

Reply:
ohh ok i get it. I'll change it. Thanks!